src/getDiffPixel.js

```diff
@@ -1,20 +1,11 @@
 const compose = require('./compose');
-const euclideanDistance = require('./euclideanDistance');
+const colorDelta = require('./colorDelta');
 
 const TRANSPARENT = [0, 0, 0, 0];
 
 module.exports = function getDiffPixel(previousPixel, currentPixel) {
   // Compute a score that represents the difference between 2 pixels
-  //
-  // This method simply takes the Euclidean distance between the RGBA channels
-  // of 2 colors over the maximum possible Euclidean distance. This gives us a
-  // percentage of how different the two colors are.
-  //
-  // Although it would be more perceptually accurate to calculate a proper
-  // Delta E in Lab colorspace, we probably don't need perceptual accuracy for
-  // this application, and it is nice to avoid the overhead of converting RGBA
-  // to Lab.
```

Comment:
Did you profile the new algorithm to figure out what the cost is in terms of performance?

Reply:
I did not

Reply:
I would think that the majority of time is still spent in lcs land, + hashing the rows of pixels. But it might be good to at least get some numbers. 

How about a comparison between
```js
for (let i = 0; i < 1000000; i++) {
  colorDelta([12, 12, 12, 255], [5, 7, 8, 255]);
}
```
and
```js
for (let i = 0; i < 1000000; i++) {
  euclideanDistance([12, 12, 12, 255], [5, 7, 8, 255]);
}
```
?

Reply:
I used this test script (swapping out the module to test the two different implementations):

```js
const fn = require('./src/euclideanDistance');

for (let i = 0; i < 1000000; i++) {
  fn([Math.floor(Math.random() * 255), 12, 12, 255], [5, 7, 8, 255]);
}
```

I timed it by running it via `time` in my shell.

node 8.9.1

euclideanDistance: 0.07s user 0.01s system 97% cpu 0.089 total
colordelta: 0.14s user 0.02s system 101% cpu 0.156 total

node 12.13.0

euclideanDistance: 0.05s user 0.01s system 98% cpu 0.065 total
colorDelta: 0.09s user 0.01s system 102% cpu 0.101 total

Reply:
Nice! So on a 1000x1000px image, this only adds at most 50ms. Right? That’s certainly 👍

Reply:
I modified a conditional to improve perf by moving the alpha comparison after the rgb comparisons for the fast path exit:

```diff
-  if (a1 === a2 && r1 === r2 && g1 === g2 && b1 === b2) {
+  if (r1 === r2 && g1 === g2 && b1 === b2 && a1 === a2) {
```

This took off ~0.01s from the benchmark.